### DISCUSSION TITLE
Metadata providers

### DISCUSSION THREAD

**Discussion by PaoloVittori** [music-assistant/support#1408] [Q&A] [answered]
Is there a way to disable metadata providers?
I use mp3tag to set metadata content. Metadata providers give an incorrect result.

**Comment by marcelveldt** [comment]
I think you jump into conclusions too fast. Metadata providers are only used to complete metadata if not already present. I think you should first describe your issue

**Reply by PaoloVittori** [reply]
I usually set  metadata  using mp3tag (title, song, author etc) for all my music files (stored on NAS).
When I synchronize the NAS folder, the metadata shown by the music assistant integration are often different from those set with mp3tag.

**Reply by OzGav** [reply]
Please provide s specific example with screenshots. Also by the sounds of it you aren't providing sufficient tag information in your tracks for MA to work best. We recommend you use Picard to comprehensively tag all your tracks

**Comment by marcelveldt** [comment, marked as answer]
> I usually set metadata using mp3tag (title, song, author etc) for all my music files (stored on NAS). When I synchronize the NAS folder, the metadata shown by the music assistant integration are often different from those set with mp3tag.

That sounds like a bug and should really NOT be possible. Tags within files itself always are leading. I did fix a lot of bugs in this regard in the last couple of beta's so maybe you're just running into an old bug here. Maybe try a fresh install of the beta67 (which is the current latest) to see if this still happens ?

if so, please share some screenshots of the tags within Mp3Tag and the result in MA as I'm really wondered why this could happen. Like I said, metadata providers are ONLY used to complete missing metadata such as images but may never ever overwrite existing info.

Also, Like @OzGav said, Mp3tag is a bit limited tool, the official Musicbrainz Picard tool is way more powerful and will also set the more advanced tags out of the box like musicbrainz id's, album type etc.
